openedx/core/djangoapps/django_comment_common/comment_client/course.py

```diff
@@ -39,3 +40,50 @@ def get_course_commentable_counts(course_key: CourseKey) -> Dict[str, Dict[str,
         metric_action='commentable_stats.retrieve',
     )
     return response
+
+
+@function_trace("get_course_user_stats")
+def get_course_user_stats(course_key: CourseKey, params: Optional[Dict] = None) -> Dict[str, Dict[str, int]]:
```

Comment:
```suggestion
def get_course_user_stats(course_key: CourseKey, params: Optional[Dict] = {}) -> Dict[str, Dict[str, int]]:
```

Reply:
@AhtishamShahid  This is  an anti-pattern. You should never set a mutable default argument for reasons described here: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments